nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h

```diff
@@ -6,10 +6,13 @@
  */
 
 /** @file
- * @brief nRF70 DK TX power ceiling configuration.
+ * @brief nRF7002 DK TX power ceiling configuration.
```

Comment:
it was for both nRF7002/nRF7001 DKs

Reply:
There is only the 7002 DK - the others are emulators